components/inputs/input-date-range.js

```diff
@@ -106,6 +111,42 @@ class InputDateRange extends FormElementMixin(RtlMixin(LocalizeCoreElement(LitEl
 		if (!this.label) {
 			console.warn('d2l-input-date-range component requires label text');
 		}
+
+		if (this.required && (!this.startValue || !this.endValue)) {
+			if (this.startValue) {
+				const endInputDate = new Date(this.startValue);
+				endInputDate.setDate(endInputDate.getDate() + 1);
+				const parsedObject = parseISODate(endInputDate.toISOString());
+				this.endValue = formatDateInISO(parsedObject);
```

Comment:
I guess we're kind of assuming that if they set a start date, that both it and the day after are "in range"? Things get really complicated if the max & min only span 1 day. 😕

Reply:
Yup exactly - it seemed unnecessarily complicated to consider/handle other cases especially since they're fairly unlikely.